src/app/manager-dashboard/certifications/certifications.component.html

```diff
@@ -18,8 +18,8 @@
   <div *ngIf="!emptyData; else notFoundMessage" class="view-container view-full-height view-table">
     <mat-table #table [dataSource]="certifications" matSort [matSortDisableClear]="true">
       <ng-container matColumnDef="name">
-        <mat-header-cell *matHeaderCellDef mat-sort-header>Certification Name</mat-header-cell>
-        <mat-cell *matCellDef="let element">{{element.name}}</mat-cell>
+        <mat-header-cell i18n *matHeaderCellDef mat-sort-header>Certification Name</mat-header-cell>
+        <mat-cell i18n *matCellDef="let element">{{element.name}}</mat-cell>
```

Comment:
`i18n` translation will not work for interpolation.  You should remove the one on this line.

Reply:
Will do, should I keep it in the line 21?

Reply:
Line 21 is good.  It's just the `{{`/`}}` where the value of a component property is used for text that cannot be translated with i18n